### DISCUSSION TITLE
Upgrading from 1.4 to 1.8

### DISCUSSION THREAD

**Discussion by weirdyang** [OrchardCMS/OrchardCore#16285] [Q&A] [answered]
Hi,

My current OrchardCore instance is on 1.4 and .NET 6, and I would need to do an upgrade to 1.8 and .NET 8.

My current plan is to first update the framework to .NET 8, test and ensure there's no breaking changes, then proceed to do the update for OrchardCore 1.4 to 1.8.

Are there any known issues for .NET 8 compatibility with OrchardCore 1.4?

Thanks

**Comment by hishamco** [comment]
If I remember `1.8.0` is the first release that target .NET 8.0

**Comment by Piedone** [comment, marked as answer]
The recommended approach is go version by version, following the release notes, so start with 1.5: https://docs.orchardcore.net/en/latest/releases/1.5.0/ Trying to jump ahead either in the OC version or the .NET one will make things a lot harder.
